src/components/atoms/Badge/Badge.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import styled from 'styled-components'
+
+// T Y P E S
+
+export interface BadgeProps {
+  value: string | number
+  className?: string
+}
+
+// C O M P O N E N T S
+
+export const Badge: React.FC<BadgeProps> = (props) => {
+  const { value, className = null } = props
```

Comment:
```suggestion
  const { value, className } = props
```

It's better that `className` still undefined

Reply:
if `className` is `undefined`, then it will show `class="undefined"` in browser.
`null` value removes the `class` attribute in browser, doesn't it?